### PR TITLE
Added page listing to timeline

### LINKED ISSUE #226: Timeline: pages listing

### PR DESCRIPTION
**Pull request by lauriLukkarinen**
Resolves #226

### CODE REVIEW

File: src/components/content-editor/timeline-editor.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * @param props component props
+ */
+const renderDraggableItem = (page: ExhibitionPage, props: Props) => {
+  const { onClick, selectedPage, classes } = props;
+  const selected = selectedPage?.id === page.id;
+  return (
+    <Draggable
+      key={ page.id }
+      axis="x"
+      grid={[5, 0]}
```
Comment (jarinykanen) [inline src/components/content-editor/timeline-editor.tsx:64]:
```suggestion
      grid={ [5, 0] }
```

File: src/components/screens/content-editor-screen.tsx [hunk 2]
```diff
@@ -381,6 +390,20 @@ class ContentEditorScreen extends React.Component<Props, State> {
...
     });
   }
+
+  /**
+   * 
+   * @param selectedPage Event handler for page click
+   * 
+   * @param selectedPage selected page
+   */
+  private onPageClick = (selectedPage: ExhibitionPage) => () => {
```
Comment (jarinykanen) [inline src/components/screens/content-editor-screen.tsx:400]:
```suggestion
  /**
   * Event handler for page click
   * 
   * @param selectedPage selected page
   */
  private onPageClick = (selectedPage: ExhibitionPage) => () => {
```

**Review by jarinykanen — CHANGES_REQUESTED** [review]
Fix comments

File: src/components/content-editor/timeline-editor.tsx (hunk 1 above)
Comment (lauriLukkarinen) [reply src/components/content-editor/timeline-editor.tsx:64]: More like this
```suggestion
      grid={[ 5, 0 ]}
```

**Pull request merged**